src/EPR.SubsidiaryBulkUpload.Application/Services/ICompaniesHouseDataProvider.cs

```diff
@@ -10,4 +10,4 @@ public interface ICompaniesHouseDataProvider
     /// <param name="subsidiaryModel">The model to populate.</param>
     /// <returns>true is the organisation model was updated; false Otherwise.</returns>
```

Comment:
true if, not is. And Otherwise doesn't need the capital O :)

Reply:
done